.goreleaser.yml

```diff
@@ -67,7 +67,6 @@ dockers:
   - image_templates:
     - "launchdarkly/ld-relay:{{ .Version }}-i386"
```

Comment:
So, this isn't exactly elegant.. but what I've done is just split out the `latest` tags from each instance of `image_templates`, into a new `image_template` block.

This is because we can only control the `skip_push` argument at the `image_template` level.

The idea is - for each architecture, built *all* the tags, but *only* push the `latest` tag for the architecture if we're on the default branch. 

Then in the manifest section starting on line 169, don't push the `latest` manifest unless we're on the default branch.